model/src/main/java/com/basistech/rosette/dm/Transliteration.java

```diff
@@ -0,0 +1,202 @@
+/*
+* Copyright 2014 Basis Technology Corp.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+package com.basistech.rosette.dm;
+
+import com.google.common.base.Objects;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+public final class Transliteration extends BaseAttribute {
+
+    private Map<String, String> scriptMap;
+
+    protected Transliteration(Map<String, String> scriptMap,
+                              Map<String, Object> extendedAttributes) {
+        super(extendedAttributes);
+        this.scriptMap = ImmutableMap.copyOf(scriptMap);
+    }
+
+    /**
+     * Gets the transliterated text in the given script
+     * @param script The script to look for.
+     * @return The transliterated text, or {@code null} if no mapping was found.
+     */
+    public String getWithScript(String script) {
```

Comment:
I think we should standardize on `ISO15924` rather than String for the script code